public/view/homework.php

```diff
@@ -391,56 +411,49 @@ function version_changed(){
   }
   echo '</select>';
   echo '</form>';
-  echo '</div>'; // left half of split row
 
   // -----------------------------------------------------
   // CHANGE ACTIVE VERSION & CANCEL SUBMISSION BUTTONS
 
-  echo '<div class="split-row" style="margin-left: 15px;">';  // right half of split row
-
   if ($assignment_version != $active_version) {
     echo '&nbsp;&nbsp;&nbsp;&nbsp;';
     echo '<form class="form_submit" action="?page=update&semester='.$semester.'&course='.$course.'&assignment_id='.$assignment_id.'&assignment_version='.$assignment_version.'" method="POST"';
     echo 'onsubmit="return check_version_change()"';
     echo'>';
     echo '<input type="hidden" name="csrf_token" value="'.$_SESSION['csrf'].'"" />';
-    echo '<input type="submit" class="btn btn-primary" value="Set Version '.$assignment_version.' as ACTIVE Submission Version"></input>';
+    echo '<input type="submit" class="btn btn-primary" value="Make Version '.$assignment_version.' ACTIVE (Grade This Version)"></input>';
     echo '</form>';
   }
 
-  if ($active_version != 0) {
+  if ($active_version != 0 && $assignment_version == $active_version) {
```

Comment:
I think the previous "Cancel Submission" button (now the "Do Not Grade This Version(...)") should be present in every version other than version 0, and it should have text "Do Not Grade This Assignment" or something similar.